sdk/keyvault/azure-keyvault-secrets/samples/README.md

```diff
@@ -4,6 +4,7 @@ languages:
   - python
 products:
   - azure
+  - azure-key-vault
```

Comment:
(not knowing where this metadata feeds into) Not `azure-keyvault-secrets` ?

Reply:
it's for docs. they don't have sdk based tags, but product based

Reply:
OK, I actually searched for comparators and found "azure-storage-blob" [here](https://github.com/Azure/azure-sdk-for-python/blob/a59b39a1bc11fe8556c94df26a71aa7ebf7d9d8f/sdk/storage/azure-storage-blob/samples/README.md), so I thought it might be analogous.

Reply:
yes, that one is wrong :) I submitted a PR to fix that: https://github.com/Azure/azure-sdk-for-python/pull/8648